core/embed/trezorhal/stm32f4/usb/usbd.c

```diff
@@ -95,6 +100,8 @@ static secbool __wur check_desc_str(const char *s) {
 void usb_init(const usb_dev_info_t *dev_info) {
```

Comment:
should we return (sec)bool? even though this seems to be infallible, it would be more consistent with the rest of the drivers

Reply:
see this change 5c362db1cd8169f39a83b593480bd5ab66e39a9f and the comment above